src/molecules/molecule.js

```
@@ -102,22 +98,43 @@ export default class Molecule extends Atom {
+    if (childCount === 0) {
+      return [1, 1]; // be nice about division by 0
+    }
+    switch (this.getState().status) {
+      case Status.READY:
+        return [childCount, childCount];
+      case Status.WAITING:
+        return [0, childCount];
+      case Status.PROCESSING:
+        const readyChildCount = this.nodesOnTheScreen.filter(
+          (node) => node.getState().status === Status.READY
+        ).length;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)